### PR TITLE
updating the CSS and generating version 0.8

### PR DESCRIPTION
**Pull request by luizfaias**

### CODE REVIEW

**Comment by skipjac** [comment]
Why all the changes to the templates? Was this a ask from a customer?

**Comment by dpawluk** [comment]
Did I break something?

**Comment by skipjac** [comment]
@dpawluk  I don't think so most of the changes are cosmetic but it looks like @luizfaias removed ticket forms which I need to understand why

**Comment by luizfaias** [comment]
@skipjac and @dpawluk I did create a custom version of the app for a customer (Farfetch) without ticket forms but I haven't removed them from the official version here

no new or removed functionality here, just cosmetic changes really

screenshot:
project_app_new_version

File: app.js [hunk 1]
```diff
@@ -225,17 +226,11 @@
         };
       }
     },
-    //end requests
+
     init: function() {
       this.getTicketFormData(1);
       this.getProjectData();
     },
-    // presetTemplate: function(){
```
Comment (skipjac) [inline app.js]: you are going to have to add these back in. They are used

File: app.js [hunk 2]
```diff
@@ -225,17 +226,11 @@
...
-    //   var settings = JSON.parse(this.settings.settingsMap);
```
Comment (skipjac) [inline app.js]: same goes here. they are commented out but people do use them and uncomment them

File: app.js [hunk 3]
```diff
@@ -530,21 +523,23 @@
         var selectedFormArray = this.ticketForms[this.$('#zendeskForm').val()];
         this.ticketFieldObj.forEach(function(d){
           if(_.contains(selectedFormArray, d.id)){
-            this.displayFields.push(d);
+            if(d.type != "tickettype" && d.type != "priority") {
```
Comment (skipjac) [inline app.js]: Why the if statement?

File: templates/multicreate.hdbs [hunk 4]
```diff
@@ -1,82 +1,79 @@
-<section data-main>
- <!-- <div id='presets'>  can be used for preset templates
```
Comment (skipjac) [inline templates/multicreate.hdbs]: this is templates can add for preset tickets.

File: templates/requester.hdbs [hunk 5]
```diff
@@ -1,79 +1,76 @@
...
+    </div>
+
+    <div class="control-group" style="display: none;">
+      <label for="zendeskSelect">Group Id</label>
+      <input type="text" required placeholder="Group id" name="zendeskSelect" id="zendeskSelect" value=""/>
+    </div>
+
+    <div class="control-group">
       <div class="select" required = "required">
-          <label for="zendeskForm">Ticket Form</label>
```
Comment (skipjac) [inline templates/requester.hdbs]: not seeing the ticket form in template.

**Pull request merged**